com.sap.cloud.lm.sl.cf.core/src/test/java/com/sap/cloud/lm/sl/cf/core/util/NameUtilTest.java

```diff
@@ -17,38 +17,6 @@
 
 public class NameUtilTest {
 
-    @Test
-    public void testIsValidXsAppNameWhenNamesAreValid() {
```

Comment:
Because the method "isNameValid" did nothing custom, it just delegated its arguments to String::matches. That's why it's deleted and so are its tests.